src/pages/LoginView.jsx

```diff
@@ -79,9 +85,8 @@ function LoginView(props) {
    * Render
    ************************************/
 
-  return isLoggedIn ? (
-    <Redirect to={props.location.state?.referer || '/'} />
-  ) : (
+  if (user) return <Redirect to='/' />;
```

Comment:
I see what you mean, but I think it would be best to make it impossible for an authenticated user to go to Login. The only way an authenticated user should be able to get to Login is if they type it in the address bar, and this check just dumps them to home if they do that. 

We could have a wrapper like ProtectedRoute, but I see the same behavior only with Register. I'm not sure if it's worthwhile dedicating a component to that. What do you think?

Reply:
Ah I see the issue. This is a wrapper for the Login so it can only have one functionality, ignore my comments. Just misunderstood the Protected Route